pyproject.toml

```diff
@@ -15,14 +15,18 @@ classifiers = [
   "Programming Language :: Python :: 3",
 ]
 
-requires-python = "~=3.8"
+requires-python = "~=3.9"
 dependencies = [
-  "numpy",
-  "matplotlib",
-  "colorspacious",
-  "scipy",
+  "numpy ~=1.22",
+  "matplotlib ~=3.5",
+  "colorspacious ~=1.1",
+  "scipy ~=1.8",
 ]
 
+[project.optional-dependencies]
+PySide = ["PySide2 ~=5.15"]
+PyQt = ["PyQt5 ~=5.15"]
```

Comment:
I think these used to be called "extras" instead of optional dependencies? 

When installing, behavior is same as before (you won't get Qt), unless someone specifies `viscm[PySide]` to bring in Qt bindings.